lix/README.md

```diff
@@ -1,11 +1,10 @@
 # Lix (previously Project Lisa)
 
-Lix is the git compatible data layer for inlang apps.
+Lix is the git compatible data storage for applications with an "edit > review > automate" workflow.
 
-We build lix to bring developers and non technical editors together without sacrificing the power of full version control.
+We build lix to bring developers and non technical editors together without sacrificing the power of full change control.
 
-We will also support other usecases for many applications with a
-"create > review > automate" flow:
+"Change control" is like version control, but giving apps the options to stay out of the way, be faster and also support more complex formats and derived assets.
```

Comment:
@janfjohannes the explanation of change control is "fluffy". what does "stay out of the way mean?"

@janfjohannes @felixhaeberle is this (first iteration of the) explanation better (not fluffy)?

> Change control is version control with a semantic understanding of changes. 
> 
> For example, change control allows querying changes in a data structure like "Who changed translation 'foo' in the file 'translations.json'?"

Reply:
The examples can be extended ofc. 

- semantic querying 
- automations based on "this translation changed' and co suddenly become easy [because of semantic querying]

Reply:
hmm dont think the "semantic understanding" is fundamental, its the how not the what or why, something required to implement change control, but utlimately the goal is to avoid having to do manual steps while working on changing data. on the other hand we dont sacrifice control of having consistent data. that is what i meant with power of full change control "without gettgin in your way".

Reply:
>  its the how not the what or why 

How data changes is semantic understanding for me. Anyhow, let's hop on a call on Monday. This part of the readme is not clear.